samples/projects/java-project/gradle

```diff
@@ -0,0 +1 @@
+../../../gradle
```

Comment:
Is this a symlink? If so, it's not going to work in vanilla Windows.

Reply:
You're correct, I tested it on my local machine and it worked, Windows doesn't seem to pull the symlinks correctly (even though I used windows to make them). I've changed the scripts to invoke the top-level gradle wrapper.